### PR TITLE
.github: Trigger daily riscv64 snapd edge builds

### PR DESCRIPTION
**Pull request by xnox**
Use github actions workflow to trigger daily riscv64 snapd edge
builds.

### CODE REVIEW

**Review by mvo5 — APPROVED** [review]
Thank you

**Comment by mvo5** [comment]
This looks nice, I would like to have a quick check from @sergiocazzolato  too before merging

**Review by sergiocazzolato — APPROVED** [review]
Nice, thanks

File: .github/workflows/riscv64-builds.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: lp-snap-request-build-riscv64
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+  workflow_dispatch:
```
Comment (sergiocazzolato) [inline .github/workflows/riscv64-builds.yml:6]: Is it needed?

**Review by sergiocazzolato — APPROVED** [review]

**Pull request merged**